### PR TITLE
Feat/markdown styles issue 44

### LINKED ISSUE #44: determine priority and proper rendering of a hierarchy of styles
E.g.

# Renders `code` with title styles

vs we do not do this:

Screen Shot 2020-01-02 at 3 20 30 PM

**Other examples:**

> a **bold** or _italic_ inside a quote

### PR DESCRIPTION
**Pull request by frontexpert**
Fixes #44 
Hello, @shcheklein. I just fixed the markdown styles issue  - #44 
It seems that it work as well in my side now.
Screenshot at Jan 24 23-18-23
Screenshot at Jan 24 22-02-01
Screenshot at Jan 24 22-41-41

Please check my result and let me know your feedback

Thanks.

### CODE REVIEW

**Comment by shcheklein** [comment]
@frontexpert could you make a screenshot "before" with the same text, please?

**Comment by frontexpert** [comment]
Sure

**Comment by frontexpert** [comment]
Screenshot at Jan 24 23-18-23

**Comment by shcheklein** [comment]
do we inherit code in quotes?

do we want to invert bold in titles for example if I specify bold (since they are already bold)? I think that's how Medium (dev.to?) behave?

**Comment by frontexpert** [comment]
Here is the screenshots "before" with the same text

Screenshot at Jan 24 23-50-17
Screenshot at Jan 24 23-39-48
Screenshot at Jan 24 23-46-40

**Comment by frontexpert** [comment]
> do we inherit code in quotes?
Yes, as you can see my screenshots, code style will be inherited from upper layer styles.

**Comment by frontexpert** [comment]
> do we want to invert bold in titles for example if I specify bold (since they are already bold)? I think that's how Medium (dev.to?) behave?

For the problem, I updated a bit more.
Please take a look at this screenshot.
Screenshot at Jan 25 00-36-10

**Comment by frontexpert** [comment]
@shcheklein I followed the dev.to styles.
Here is my progress.

**Comment by frontexpert** [comment]
@shcheklein 
I checked dev.to and medium in detail. After that, I followed dev.co styles and pushed my result. Once you are available, I want you to check the edge cases by yourself.
Thanks.

File: src/components/markdown/styles.module.css [hunk 1]
```diff
@@ -130,8 +113,8 @@
   blockquote:not(:global(.twitter-tweet)) {
     margin: 30px 0;
     padding-left: 25px;
-    font-style: italic;
     position: relative;
+    font-size: 0.92em;
```
Comment (shcheklein) [inline src/components/markdown/styles.module.css]: where do we get these numbers from? are there constants that define this and we can reuse them with `var`?

**Comment by shcheklein** [comment]
@frontexpert could summarize and show screenshots of what specific situations you have changed? Also see my comment please.

File: src/components/markdown/styles.module.css (hunk 1 above)
Comment (frontexpert) [reply src/components/markdown/styles.module.css]: 0.92em was from dev.co style. Dev.co is using 0.92em font size for the quote. em unit is related to the font size of the parent container. If parent container have 16px font size, quote will have 0.92em(=14.72px), if parent container have 24px font size, quote will have 22.08px. And the font-size of the parent container is setting as responsive size. 
Just like this:
  @media (--md-scr) {
    @mixin text-mobile;
  }

  @media (--lg-scr) {
    @mixin text-desktop;
  }  
Screenshot at Jan 27 03-02-42

**Comment by frontexpert** [comment]
@shcheklein 
Sorry for missing issues in my side.
Can you kindly take a look at my result again?
https://deploy-preview-81--dvc-blog.netlify.com/january-20-community-gems

different_styles
code_style

If you have any other problems, please let me know.
Thanks.

**Comment by frontexpert** [comment]
@shcheklein I already reverted quote style - italic.
You can check the result on https://preview-franc-blog.netlify.com/january-20-community-gems

and Can you assign me more tasks?

**Pull request merged**